checkout.html

```diff
@@ -1,118 +1,175 @@
+                </div>
+                <div class="input-control">
+                    <input type="text" id="lastName" placeholder="LAST NAME .." />
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+
+                <label for="Street Address">STREET ADDRESS</label>
+                <div class="input-control twoColumns">
+                    <input type="text" id="address" placeholder="STREET ADDRESS .." />
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+
+
+                <label for="City">CITY</label>
+                <div class="input-control">
+                    <select name="city" class="cities" id="cityId">
+                        <option value="">Select City</option>
+                    </select>
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div class="hide"></div>
+
+                <label for="State">STATE</label>
+                <div class="input-control">
+                    <select name="state" class="states" id="stateId">
+                        <option value="">Select State</option>
+                    </select>
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div class="hide"></div>
+
+                <label for="country">COUNTRY</label>
+                <div class="input-control">
+                    <select name="country" class="countries" id="countryId">
+                        <option value="">Select Country</option>
+                    </select>
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div class="hide"></div>
+
+                <label for="Postal code">POSTAL CODE</label>
+                <div class="input-control">
+                    <input type="text" id="postCode" placeholder="POSTAL CODE .." />
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div></div>
+
+                <label for="phone">PHONE</label>
+                <div class="input-control">
+                    <input type="text" id="phone" placeholder="PHONE .." />
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
```

Comment:
I notice that around tablet size some of the text in the form get cut off, may have to fix the styling on this later.
Also, it feels weird being able to open state and city select box but not being able to pick anything from them, may want to disable state and city before the preceding one is picked.
Example use case: 
User start with only being able to select country while state and city is disabled (or maybe hidden).
When user select the country then state becomes enabled, if user deselect the country state will revert to default and be disabled.
When user select the state then the city becomes enabled, if anything preceding it is reverted to default it becomes disabled.

Reply:
1) created the issue "Order in displaying state, city, country in checkout page #53"
2) if I could not solve the issue of cutting off texts tonight,  I will create a new issue for it.